### PR TITLE
the rewrite

### PR DESCRIPTION
**Pull request by SrIzan10**
this is automata v2, which is ready to be merged!

### CODE REVIEW

**Comment by jacoobes** [comment]
cna you add tests?

**Comment by SrIzan10** [comment]
I think I should, but I'll just add it as a linear issue for now

**Comment by jacoobes** [comment]
is your .yarn  set up correctly? i see some things that seem like they wouuldnt belong?
.gitignore

**Comment by jacoobes** [comment]
sorry i mean .gitignore

**Comment by SrIzan10** [comment]
oh, I think it's fine right? https://github.com/sern-handler/automata/blob/feat/the-rewrite/.gitignore
I recommend reviewing from the branch's files instead of the diff, all the previous code got deleted

File: src/index.ts [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+process.stdin.on('data', async (data) => {
+  if (data.toString().trim() === 'test thing') {
+    console.log('Testing thing')
+    // leaving for other testing purposes
+  }
+})
+
+async function react(owner: string, repo: string, commentId: number, reaction: Reaction) {
+  return await octokit.request('POST /repos/{owner}/{repo}/issues/comments/{comment_id}/reactions', {
```
Comment (jacoobes) [inline src/index.ts]:
```suggestion
  return octokit.request('POST /repos/{owner}/{repo}/issues/comments/{comment_id}/reactions', {
```

File: src/index.ts [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+      })
+      break;
+  }
+
+  return c.json({ ok: true })
+})
+
+app.post('/ev/updateDocsJson', async (c) => {
+  const validate = await validateJsonWebhook(c)
+	if (!validate)
```
Comment (jacoobes) [inline src/index.ts]: weird indenting

**Review by jacoobes — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 2 above)
Comment (SrIzan10) [reply src/index.ts]: huh

**Comment by jacoobes** [comment]
clean, im liking the direction of this!

**Pull request merged**